templates/products/_product-navigation.html

```diff
@@ -0,0 +1,14 @@
+<div class="col-3 col-medium-2 u-hide--small">
```

Comment:
Not sure why this is set to hide on small screens? It's not possible to navigate between the categories without this.
```suggestion
<div class="col-3 col-medium-2">
```